ui/pages/Project/components/FamilyPage.jsx

```diff
@@ -61,11 +61,11 @@ const BaseVariantDetail = ({ project, family, hasActiveVariantSample, compact, g
         <VerticalSpacer height={10} />
         <CreateVariantButton family={family} />
         <VerticalSpacer height={10} />
-        {project.isMmeEnabled &&
+        {/* {project.isMmeEnabled &&
```

Comment:
I guess alternatively we could just set this project property to default-disabled instead of enabled?